database/seeders/DatabaseSeeder.php

```diff
@@ -12,11 +15,10 @@ class DatabaseSeeder extends Seeder
      */
     public function run(): void
     {
-        \App\Models\User::factory()->create([
+        User::factory()->create([
             'name' => 'admin',
-            'username' => 'admin',
-            'email' => env('ADMIN_EMAIL'),
-            'password' => env('ADMIN_PASSWORD'),
+            'email' => config('admin.email'),
+            'password' => Hash::make(config('admin.password')),
```

Comment:
Je suis pas sur que tu doive faire "Hash::make" car tu as déjà cast "password" en "Hashed"

Reply:
I do not know how you see this kind of things but yes, I removed `Hash `Facade and it worked